### PR TITLE
Albanian Translation

### PR DESCRIPTION
**Pull request by gertiksxx**

### CODE REVIEW

File: vft505qa.qmb.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿Albanian Lang (not complete)
+
+Signed-off-by: gertpere <gert.pere@gmail.com>
```
Comment (zuphilip) [inline vft505qa.qmb.txt]: I guess you tried to sign off your commit message. Somehow there is now this additional file, which we have to delete again.

**Comment by zuphilip** [comment]
@stweil How to proceed now?

**Comment by stweil** [comment]
Talk to the author?

File: locale/al_AL.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+# Gert Pere <gpere@mail.uni-mannheim.de>, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PalMA 0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-01-04 15:25+0100\n"
+"PO-Revision-Date: 2014-04-09 21:31+0200\n"
+"Last-Translator: Gert Pere <gpere@mail.uni-mannheim.de>"
+"Language-Team: Albanian <en@li.org>\n"
```
Comment (zuphilip) [inline locale/al_AL.UTF-8/LC_MESSAGES/palma.po]: This should then be
```
<al@li.org>\n
```

**Comment by kba** [comment]
Here's a rebase with just the translation and a signed-off commit message: https://github.com/kba/PalMA/tree/albanian-rebase

**Comment by stweil** [comment]
Hello @gertiksxx, 

I'd like to add your translation to the PalMA code, but before doing so, several issues have to be fixed:

* The commit message needs your signature (last line must look like `Signed-off-by: Gert Pere <gpere@mail.uni-mannheim.de>`).

* File `vft505qa.qmb.txt` must be removed from the pull request.

* The single commits should be combined ("squashed") into one commit.

Could we meet to fix that together, or should I add the signature for you and fix also the other two issues?

**Comment by gertiksxx** [comment]
Dear Stefan,

Thanks for the e-mail. I can come to the university, but if there is no
problem you can fix the issues by yourself as well.

Kind regards
Gert

**Comment by stweil** [comment]
@gertiksxx, then I'll fix and apply the pull request as suggested. Which e-mail address should I use for your signature? Your gmail address or the uni-mannheim address?

**Comment by stweil** [comment]
@gertiksxx, if you don't mind I'll use your uni-mannheim e-mail address for the signature.

**Comment by gertiksxx** [comment]
Ok, no problem

**Comment by stweil** [comment]
Thanks, this pull request with modifications as discussed is now integrated in PalMA, see commit 510d6d257dc6d004180838d107bc0731684d4c62.

**Pull request closed without merging**